server/traces/otel_http_converter.go

```diff
@@ -61,12 +69,14 @@ func convertHttpOtelSpanIntoSpan(span *httpSpan) *model.Span {
 
 	var startTime, endTime time.Time
 
-	if span.GetStartTimeUnixNano() != 0 {
-		startTime = time.Unix(0, int64(span.GetStartTimeUnixNano()))
+	if span.StartTimeUnixNano != "" {
+		startTimeNs, _ := strconv.ParseInt(span.StartTimeUnixNano, 10, 64)
+		startTime = time.Unix(0, startTimeNs)
```

Comment:
Same problem, the value for StartTimeUnixNano is a string instead of int, so we have to get it as a string and convert it by hand here.